azure-pipelines.yml

```diff
@@ -35,7 +35,13 @@ jobs:
     vmImage: ubuntu-16.04
   steps:
     - bash: |
-        "$PWD/bin/brew" test-bot
+        OLDPWD="$PWD"
+        HOMEBREW_REPOSITORY=/home/linuxbrew/.linuxbrew
+        sudo mkdir -p /home/linuxbrew
+        sudo mv "$PWD" "$HOMEBREW_REPOSITORY"
+        "$HOMEBREW_REPOSITORY/bin/brew" test-bot
+        sudo mkdir -p "$OLDPWD"
+        cd "$OLDPWD"
```

Comment:
I suggest…
```sh
	        HOMEBREW_REPOSITORY=/home/linuxbrew/.linuxbrew
	        sudo mkdir -p /home/linuxbrew
	        sudo mv "$PWD" "$HOMEBREW_REPOSITORY"
	        mkdir -p "$PWD"
	        "$HOMEBREW_REPOSITORY/bin/brew" test-bot
```

Reply:
A symlink would be preferable.

Reply:
So instead of the mv do an ln -s and mkdir the parent dir if necessary. You’ll also need to update the brew-test-bot repo similarly.

Reply:
I don't think this will work since [`bin/brew` resolves symlinks](https://github.com/Homebrew/brew/blob/fdffb395146d054b24947d127b6a1d25573473ce/bin/brew#L22-L24). I tried for a few hours yesterday to get it to work using the linking method that you suggest and gave up.

Steps to reproduce on macOS:

```
cd
ln -s /usr/local/ prefix
cd prefix
bin/brew config
```

The output will say `HOMEBREW_PREFIX: /usr/local` but should say `~/prefix/`

Reply:
Thanks for trying. In that case you could move the directory and symlink it back to the original location.